app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Elf, Dwarf]) -> int:
+    return sum(player.get_rating() for player in players_list)
+
+
+def elves_concert(elves_list: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in elves_list]
```

Comment:
You don't use list which is created by comprehension here, so it is just a memory waste. Fix everywhere